contracts/WidoCollateralSwap.sol

```diff
@@ -64,19 +64,18 @@ contract WidoCollateralSwap is IERC3156FlashBorrower, IFlashLoanSimpleReceiver {
         POOL = IPool(_addressProvider.getPool());
     }
 
-    /// @notice Performs a collateral swap
+    /// @notice Performs a collateral swap with Equalizer
     /// @param existingCollateral The collateral currently locked in the Comet contract
     /// @param finalCollateral The final collateral desired collateral
     /// @param sigs The required signatures to allow and revoke permission to this contract
     /// @param swap The necessary data to swap one collateral for the other
     /// @param comet The address of the Comet contract to interact with
-    function swapCollateral(
+    function swapCollateralEqualizer(
```

Comment:
Is there anything Equalizer specific or it is just ERC3156? If it is just that maybe we just name it so and any ERC3156 compatible provider can be used.

Reply:
Any ERC3156 provider could be used, but since we are initializing the addresses of the providers on the constructor, we would need to send an ID on the `swapCollateralERC3156` to know which one it is.

On the other hand, we would have some like `swapCollateralEqualizer` and `swapCollateralEuler`, each using it's hardcoded address.

On why I set the addresses on the constructor:
The interface of `IFlashLoanSimpleReceiver` from Aave requires the contract to have those public methods that return the addresses, and for that we can only send initialize it at construction.

We could have Aave hardcoded and ERC3156 dynamic... but I prefered to have a clear convention intially. We could change it tho.